src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -381,23 +322,45 @@ export function getSeriesItemData(
     });
 }
 
-export interface ISeriesItemConfig {
-    color: string;
-    legendIndex: number;
-    data?: any;
-    name?: string;
-    yAxis?: number;
-    xAxis?: number;
-}
-
 export function getHeatmapSeries(
     executionResultData: Execution.DataValue[][],
     measureGroup: Execution.IMeasureGroupHeader["measureGroupHeader"],
 ) {
-    const data = [] as any;
-    executionResultData.forEach((rowItem: any, rowItemIndex: number) => {
-        rowItem.forEach((columnItem: any, columnItemIndex: number) => {
-            data.push({ x: columnItemIndex, y: rowItemIndex, value: parseValue(columnItem) });
+    const data: IPointData[] = [];
+    const nullColor: IPatternObject = {
+        pattern: {
+            path: {
+                d: "M 10 0 L 0 10 M 9 11 L 11 9 M 4 11 L 11 4 M -1 1 L 1 -1 M -1 6 L 6 -1",
+                stroke: GRAY,
+                strokeWidth: 1,
+                fill: WHITE,
+            },
+            width: 10,
+            height: 10,
+        },
+    };
+    let value: number;
```

Comment:
same here define as const on first usage

Reply:
Same. i tried to avoid declare variable in loop forEach.

Reply:
done